tests/integration/server/users_api_tests.py

```diff
@@ -50,3 +53,25 @@ def test_invalid_duplicate_username():
         assert response.json()["message"] == expected["message"]
         assert response.json()["code"] == expected["code"]
         delete_user_by_username(user_input["username"])
+
+
+def test_syncdirectionflag_username():
+    """ Testing the presence and the value of syncdirection flag  is set to OUTBOUND of a user in users table. """
```

Comment:
Same here.

Reply:
But both the Linter & Formatter successfully 100% passed.

Reply:
Linter and formatter doesn't look at doc strings, I forgot why docstrings weren't formatted, but according to PEP8 standards, docstrings should be no longer than 72 characters:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
I get it, but all these rules should be incorporated into our LINTER & FORMATER codes. Also, in our existing code base there are several examples where you can see the doc strings length more than 85.

![image](https://user-images.githubusercontent.com/23491817/55592911-d8bd4a80-56ee-11e9-8545-1957efe1786d.png)